integration_tests/__tests__/__snapshots__/multi_project_runner.test.js.snap

```diff
@@ -10,29 +10,31 @@ Ran all test suites.
 `;
 
 exports[`can pass projects or global config 2`] = `
-"Test Suites: 2 passed, 2 total
-Tests:       2 passed, 2 total
+"Test Suites: 3 passed, 3 total
+Tests:       3 passed, 3 total
 Snapshots:   0 total
 Time:        <<REPLACED>>
-Ran all test suites in 2 projects.
+Ran all test suites in 3 projects.
 "
 `;
 
 exports[`can pass projects or global config 3`] = `
-"PASS  project1/__tests__/file1.test.js
+"PASS   BACKEND  project1/__tests__/file1.test.js
+PASS   UI  project3/__tests__/file1.test.js
 PASS  project2/__tests__/file1.test.js"
 `;
 
 exports[`can pass projects or global config 4`] = `
-"Test Suites: 2 passed, 2 total
-Tests:       2 passed, 2 total
+"Test Suites: 3 passed, 3 total
+Tests:       3 passed, 3 total
 Snapshots:   0 total
 Time:        <<REPLACED>>
-Ran all test suites in 2 projects.
+Ran all test suites in 3 projects.
 "
 `;
 
 exports[`can pass projects or global config 5`] = `
-"PASS  project1/__tests__/file1.test.js
+"PASS   BACKEND  project1/__tests__/file1.test.js
+PASS   UI  project3/__tests__/file1.test.js
```

Comment:
@aaronabramov Should it be fine because it is using `sortLines` https://github.com/facebook/jest/blob/fbd07088383e456fae050ae03feaebd40de7dbf8/integration_tests/__tests__/multi_project_runner.test.js#L34